app/main/util/date_formatter.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from flask import current_app
+
+
+def get_date_range(date_range):
+    date_from = None
+    date_to = None
+    try:
+        if date_range is not None:
+            if "date_from" in date_range:
+                dt_from = datetime.strptime(
```

Comment:
can we move the try except to only around this line as this is the only line that can cause the error

Reply:
done the changes as requested